worlds/shorthike/__init__.py

```diff
@@ -65,11 +69,18 @@ def create_items(self) -> None:
             if feather_count < 12:
                 feather_count = 12
 
-        junk = 45 - self.options.silver_feathers - feather_count - self.options.buckets
+        junk = 58 - self.options.silver_feathers - feather_count - self.options.buckets - self.options.sticks - self.options.toy_shovels
```

Comment:
Would it make sense to use `multiworld.get_unfilled_locations(player)` here instead of hardcoding the 58? (that's what I do for OW)